h2o-cve-2018-0608/mayhem/h2o/Mayhemfile

```diff
@@ -0,0 +1,13 @@
+version: '1.4'
+project: h2o-cve-2018-0608
+target: h2o
+baseimage: forallsecure/h2o-cve-2018-0608
```

Comment:
Please specify a duration (upper bound on what it would take to find, say on demo).

Reply:
ok haven't found the bug from scratch yet (ran target shortly and verified crash repro) but will update when it is found